### ISSUE TITLE
Flatpak package

### ISSUE THREAD

**Issue by espectalll** [beakerbrowser/beaker#974]
Instead of using an AppImage (which has to be manually downloaded, allocated and updated, as well as having to create a symlink by executing it - thus making it a dirty and insecure approach), or at least among it, there should be an option to install the browser from [Flatpak](https://flatpak.org/), which uses GPG-signed repositories for installing and updating, uses isolation with containers, and even works on non-Glibc Linux distros.

For more information, please refer to the developer guide (http://docs.flatpak.org/en/latest/) and, in particular, to the ["Getting Started"](http://docs.flatpak.org/en/latest/getting-started.html) section. For the delivering Beaker in particular, you could use the [Flathub](https://flathub.org/) repository.

**Comment by pmario** [comment]
For linux side I personally would prefer [snaps](https://snapcraft.io/). [Electron builder](https://www.electron.build/) seems to support this target.

**Comment by wchurch58** [comment]
Yeah a flatpak app would be awesome, I can't use beaker on Void MUSL because no electron, so a flatpak app would be perfect. AppImage kinda sucks, flatpak is much better, it kinda works like a package manager in the sense that you can update all the apps with a simple flatpak update command.

**Comment by aviwad** [comment]
It's on Flathub! I think unofficially, however...
